platform/darwin/docs/theme/assets/css/jazzy.css.scss

```diff
@@ -168,21 +173,17 @@ a {
 table {
   background: $body_background;
   width: 100%;
+  font-size: 13px;
+  line-height: 23px;
   border-collapse: collapse;
   border-spacing: 0;
   overflow: auto;
   margin: 0 0 0.85em;
 }
 
-tr {
-  &:nth-child(2n) {
-    background-color: $table_alt_row_color;
```

Comment:
You don’t like the zebra striping? 😄 I thought it rather nice, but I agree that it’s unnecessary given the horizontal rules.